migration/i7.1z/postgresql/202009111500_IDEMPIERE-3981.sql

```diff
@@ -0,0 +1,1046 @@
+ AD_IMAGE_ID, TABLEVEL, WHERECLAUSE, ORDERBYCLAUSE, COMMITWARNING, 
+ READONLYLOGIC, DISPLAYLOGIC, AD_COLUMN_ID, AD_PROCESS_ID, ISSORTTAB, 
+ ISINSERTRECORD, ISADVANCEDTAB, AD_COLUMNSORTORDER_ID, AD_COLUMNSORTYESNO_ID,
+ INCLUDED_TAB_ID, PARENT_COLUMN_ID, AD_Tab_UU, AD_Table_UU, TREEDISPLAYEDON,
+ MAXQUERYRECORDS, IsAllowAdvancedLookup, IsLookupOnlySelection)
+AS 
+SELECT t.AD_Tab_ID, t.AD_Window_ID, t.AD_Table_ID, t.Name, t.Description, 
+    t.Help, t.SeqNo, t.IsSingleRow, t.HasTree, t.IsInfoTab, tbl.ReplicationType,
+    tbl.TableName, tbl.AccessLevel, tbl.IsSecurityEnabled, tbl.IsDeleteable, 
+    tbl.IsHighVolume, tbl.IsView, cast('N' as char) AS HasAssociation, -- compatibility
+    t.IsTranslationTab, t.IsReadOnly, t.AD_Image_ID, t.TabLevel, 
+    t.WhereClause, t.OrderByClause, t.CommitWarning, t.ReadOnlyLogic, t.DisplayLogic,
+    t.AD_Column_ID, t.AD_Process_ID, t.IsSortTab, t.IsInsertRecord, t.IsAdvancedTab,
+    t.AD_ColumnSortOrder_ID, t.AD_ColumnSortYesNo_ID, t.Included_Tab_ID, t.Parent_Column_ID,
+    t.AD_Tab_UU, tbl.AD_Table_UU, t.TreeDisplayedOn, t.MaxQueryRecords,
+    t.IsAllowAdvancedLookup, t.IsLookupOnlySelection
+FROM AD_Tab t 
+	INNER JOIN AD_Table tbl ON (t.AD_Table_ID = tbl.AD_Table_ID)
+WHERE t.IsActive='Y'
+  AND tbl.IsActive='Y'
+;
+
+
+CREATE OR REPLACE VIEW AD_TAB_VT
+(AD_LANGUAGE, AD_TAB_ID, AD_WINDOW_ID, AD_TABLE_ID, NAME, 
+ DESCRIPTION, HELP, SEQNO, ISSINGLEROW, HASTREE, 
+ ISINFOTAB, REPLICATIONTYPE, TABLENAME, ACCESSLEVEL, ISSECURITYENABLED, 
+ ISDELETEABLE, ISHIGHVOLUME, ISVIEW, HASASSOCIATION, ISTRANSLATIONTAB, 
+ ISREADONLY, AD_IMAGE_ID, TABLEVEL, WHERECLAUSE, ORDERBYCLAUSE, 
+ COMMITWARNING, READONLYLOGIC, DISPLAYLOGIC, AD_COLUMN_ID, AD_PROCESS_ID, 
+ ISSORTTAB, ISINSERTRECORD, ISADVANCEDTAB, AD_COLUMNSORTORDER_ID, AD_COLUMNSORTYESNO_ID, 
+ INCLUDED_TAB_ID, PARENT_COLUMN_ID, AD_Tab_UU, AD_Table_UU, TREEDISPLAYEDON,
+ MAXQUERYRECORDS, IsAllowAdvancedLookup, IsLookupOnlySelection)
+AS 
+SELECT trl.AD_Language, t.AD_Tab_ID, t.AD_Window_ID, t.AD_Table_ID, trl.Name, trl.Description, 
+    trl.Help, t.SeqNo, t.IsSingleRow, t.HasTree, t.IsInfoTab, tbl.ReplicationType,
+    tbl.TableName, tbl.AccessLevel, tbl.IsSecurityEnabled, tbl.IsDeleteable, 
+    tbl.IsHighVolume, tbl.IsView, cast('N' as char) AS HasAssociation,
+    t.IsTranslationTab, t.IsReadOnly, t.AD_Image_ID, t.TabLevel, 
+    t.WhereClause, t.OrderByClause, trl.CommitWarning, t.ReadOnlyLogic, t.DisplayLogic,
+    t.AD_Column_ID, t.AD_Process_ID, t.IsSortTab, t.IsInsertRecord, t.IsAdvancedTab,
+    t.AD_ColumnSortOrder_ID, t.AD_ColumnSortYesNo_ID, t.Included_Tab_ID, t.Parent_Column_ID,
+    t.AD_Tab_UU, tbl.AD_Table_UU, t.TreeDisplayedOn, t.MaxQueryRecords,
+    t.IsAllowAdvancedLookup, t.IsLookupOnlySelection
+FROM AD_Tab t 
+	INNER JOIN AD_Table tbl ON (t.AD_Table_ID = tbl.AD_Table_ID)
+	INNER JOIN AD_Tab_Trl trl ON (t.AD_Tab_ID = trl.AD_Tab_ID)
+WHERE t.IsActive='Y'
+  AND tbl.IsActive='Y'
+;
+
```

Comment:
postgres script requires:
 SELECT register_migration_script('202009111500_IDEMPIERE-3981.sql') FROM dual 
same as the oracle script

Reply:
This is chilly mistake, Let me fix.